file/types.go

```diff
@@ -551,6 +556,7 @@ type FServicePackage struct {
 	Document    *FDocument        `json:"document,omitempty" yaml:"document,omitempty"`
 }
 
+// FDocument file document model
```

Comment:
```suggestion
// FDocument represents a document in Konnect. The type has been duplicated because the documents are altered before they are exported to the state file for better user experience.
```